### DISCUSSION TITLE
Copied folder and files into import directory, wife changed folder names, everything says asset offline

### DISCUSSION THREAD

**Discussion by superdong69** [immich-app/immich#9797] [Q&A]
When I look at the information of the images, the directory path does not match the current  path, as my wife changed the name.
For instance, the image says /import/folder239922unzip/image1.jpg. She unzipped an archive INTO the external library folder which had a google drive naming convention. Once unzipped, and read into the database, she changed the name of the folder.

There should be a quick and easy way to remedy this, I figured by rescanning the external library, but that does nothing. How do I quickly and easily correct this? She did this to like 5,000 images.

**Comment by DJF3** [comment]
This is one of the reasons why I have 25 external libraries. One for every year since 1999. Can't take the risk that something bad happens and I have to re-scan 500k photos :-)
When I run into a situation like this I rescan the metadata. It will then realize that the images are not available anymore (red warning box when you view the image). This should also find the new locations.
After that I "remove offline photos".

**Reply by superdong69** [reply]
Where do I find those two options?
I see remove offline photos. But what about the rescan metadata, is that just the force re-scan all data in library?
Ah I see now. After I force re-scan the entire library, under jobs, I see EXTRACT METADATA, and it's running. Once that's done, I just select remove offline photos? That won't delete any photos from the actual external library will it?
